app/controllers/avo/search_controller.rb

```diff
@@ -212,5 +204,33 @@ def search_results_count(resource)
         Avo.configuration.search_results_count
       end
     end
+
+    def parse_results(query)
+      # When using custom search services query should return an array of hashes
+      if query.is_a?(Array)
+        # Apply highlight
+        query.map do |result|
+          result[:_label] = highlight(result[:_label].to_s, CGI.escapeHTML(params[:q] || ""))
+        end
+
+        # Force count to 0 until implement an API to pass the count
+        results_count = 0
```

Comment:
Is this hidden to the user or can they see `(0)` in the search results?

Reply:
Is hidden. The count is added only when bigger than 0:

```ruby
      if results_count > 0
        header = "#{header} (#{results_count})"
      end
```